lib/service/map_constant.dart

```diff
@@ -17,3 +17,5 @@ const LatLng LATLNG_EV = LatLng(45.495668, -73.577689);
 const LatLng LATLNG_JMSB = LatLng(45.495312, -73.579033);
 const LatLng LATLNG_LOYOLA = LatLng(45.458279, -73.640436);
 const LatLng LATLNG_SP = LatLng(45.457914, -73.641557);
+
+enum DrivingMode {Car, Bus, Walk, Bike}
```

Comment:
I believe this is a redundant variable usage to the one in the global.dart that was added some PR ago. That one is an boolean list that is used for the toggle, do you can you see if you can remove one of those and keep similar usage to the same variable?

Reply:
i believe we should stick to the enum types and move that list to the GoToPage class since enum is more concrete

Reply:
Enum types will also be more efficient for equality checks

Reply:
The driving mode is saved in the DirectionNotifier class

Reply:
Ok, I will create an issue to change modify the gotopage to adjust to those changes. The rest looks good